src/inmanta/parser/plyInmantaParser.py

```diff
@@ -692,6 +692,33 @@ def p_boolean_expression_is_defined_short(p: YaccProduction) -> None:
     attach_lnr(p)
 
 
+def p_boolean_expression_is_defined_map_lookup_0(p: YaccProduction) -> None:
+    """boolean_expression : is_defined_expr IS DEFINED"""
```

Comment:
Why not just `map_lookup IS DEFINED`? Did that cause issues?

Reply:
That's a good point, I hadn't considered that. You should be able to use `map_lookup.the_map` and `map_lookup.key` though.